var/spack/repos/builtin/packages/geomodel/package.py

```diff
@@ -65,6 +65,7 @@ class Geomodel(CMakePackage):
 
     conflicts("+fullsimlight", when="+fsl", msg="FSL triggers the build of the FullSimLight")
 
+    depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
```

Comment:
Do you mind removing the `# generated comment` assuming the other has also been confirmed as being needed?